ReactNativeClient/lib/components/screens/config.js

```diff
@@ -409,7 +409,7 @@ class ConfigScreenComponent extends BaseScreenComponent {
 					</Text>
 					<View style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', flex: 1 }}>
 						<Text style={this.styles().sliderUnits}>{unitLabel}</Text>
-						<Slider key="control" style={{ flex: 1 }} step={md.step} minimumValue={md.minimum} maximumValue={md.maximum} value={value} onValueChange={value => updateSettingValue(key, value)} />
+						<Slider key="control" style={{ flex: 1 }} maximumTrackTintColor={theme.settingComponentTintColor} step={md.step} minimumValue={md.minimum} maximumValue={md.maximum} value={value} onValueChange={value => updateSettingValue(key, value)} />
```

Comment:
did you read the comment just above your change?

Reply:
Oops, sorry for that. I will remove it from the slider. But is it appropriate for the switch component ?